README.rst

```diff
@@ -60,7 +60,8 @@ Install from PyPI
 ~~~~~~~~~~~~~~~~~
 
 The latest released version may be installed from PyPI by using
-``pip``. It supports Python 2.7 or 3.6+::
+``pip``. Version 0.96 was the last version to support Python 2.7.
+Subsequent versions support Python 3.6+::
```

Comment:
Maybe say "Python 3 only", so that we don't have to remember to come back and edit this in the future? We're likely documenting our supported Python version elsewhere anyway?

Later: Actually, we don't appear to mention this separately. I still think we should drop this from here since Python support matters whether you're installing from pip or other places. Perhaps we could move this sentence before "Install from PyPI" and rephrase to say e.g. "rst2pdf support Python 3.6 or greater. rst2pdf 0.96 was the last version to support Python 2.7."

Reply:
Addressed.